modules/standard/FileSystem.chpl

```diff
@@ -164,15 +227,23 @@ proc copy(out error: syserr, src: string, dest: string, metadata: bool = false)
   }
 }
 
-/* Copies the contents and permissions of the file indicated by src into
-   the file or directory dest.  If dest is a directory, will halt with an error
-   message.  If metadata is set to true, will also copy the metadata of the
-   file to be copied.  May halt with other error messages.
-   src: the source file whose contents and permissions are to be copied
-   dest: the destination file for the contents and permissions.
-   metadata: a boolean indicating whether to copy metadata (uid, gid, time of
-             last access and time of modification) associated with the source
-             file.
+/* Copies the contents and permissions of the file indicated by ``src`` into
+   the file or directory ``dest``.  If ``dest`` is a directory, will halt with
+   an error message.  If ``metadata`` is set to ``true``, will also copy the
+   metadata (uid, gid, time of last access and time of modification) of the
+   file to be copied.
```

Comment:
You should say that in the future this command should actually copy when dest is a directory (instead of halting)